src/middleware/addUserContext.ts

```diff
@@ -6,7 +6,8 @@ const selectOrCreateUser = async (authenticationId: string) => {
 	try {
 		return await loadUserByAuthenticationId(authenticationId);
 	} catch {
-		return await createUser({ authenticationId });
+		const user = await createUser({ authenticationId });
+		return user;
```

Comment:
See https://typescript-eslint.io/rules/return-await/#in-try-catch for more info on why this is considered better.